### DISCUSSION TITLE
Upgrade from 9.02 to 9.03 - Database collation errors

### DISCUSSION THREAD

**Discussion by mp9007** [dnnsoftware/Dnn.Platform#5033] [Q&A] [answered]
Hi, when I ugrade my website from 9.02 to 9.03, I have an error of database collation.
I did not change collation of my database. 
My production environment is in Azure. I took a backup of my database from Azure and restore it in a local SQL Server.

Please see attached log files for more info
[InstallerLog20220301.txt](https://github.com/dnnsoftware/Dnn.Platform/files/8162429/InstallerLog20220301.txt)
[01.log.txt](https://github.com/dnnsoftware/Dnn.Platform/files/8162436/01.log.txt)

Also, the update info screen is stuck at 0%. No feedback.
![DNN upgrade screen](https://user-images.githubusercontent.com/38858370/156191943-6545c71b-919b-4915-ab76-fcd0096a72bf.png)

Thanks.

**Comment by bdukes** [comment, marked as answer]
It looks like your database has an inconsistent collation, do you know why that might be? Did it get changed at some point? I think you probably need to reset all of the tables/columns to use the same collation.

**Comment by mp9007** [comment]
Thanks, I manage to update the collation everywhere in my database.
